### PR TITLE
Cheapens Tobacco

### PR DESCRIPTION
**Pull request by ParadoxSpace**
Cigarettes and shit are now cheaper and more plentiful.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!review

**Comment by alsoandanswer** [comment]
No.

You _do_ realize cup noodles in the vendors are like, 20 credits?
And if we were to be a little fair, if we convert a cup noodles to about one dollar, a pack of cigarettes are about ten dollars.

Also, I've just added like, 4 new options for cigarettes, so the total amount of cigarette packets in each machine are about ~9 at most, matching what we had previously with our generic cigarettes.

This is also not including all the new chewing and rolling tobacco I just added.

**Comment by ParadoxSpace** [comment]
They do not need to be 300 dollars. Other vendor items are out of scope for this PR.

**Comment by alsoandanswer** [comment]
The reason why they're more expensive is because the better cigarettes have _more_ actual, pure nicotine.

Anyway, this price change is _way_ too low.

File: code/game/machinery/vending_types.dm [hunk 1]
```diff
@@ -345,23 +345,23 @@
...
+		/obj/item/storage/chewables/tobacco/fine = 40,
+		/obj/item/storage/fancy/chewables/tobacco/nico = 15,
 		/obj/item/storage/box/matches = 12,
 		/obj/item/flame/lighter/random = 12,
-		/obj/item/storage/cigfilters = 20,
-		/obj/item/storage/fancy/cigpaper = 32,
-		/obj/item/storage/fancy/cigpaper/fine = 48,
+		/obj/item/storage/cigfilters = 5,
+		/obj/item/storage/fancy/cigpaper = 5,
+		/obj/item/storage/fancy/cigpaper/fine = 15,
```
Comment (alsoandanswer) [inline code/game/machinery/vending_types.dm]: Why.

They weren't even that high in the first place.

File: code/game/machinery/vending_types.dm [hunk 2]
```diff
@@ -345,23 +345,23 @@
...
-		/obj/item/storage/chewables/rollable/fine = 60,
-		/obj/item/storage/chewables/rollable/nico = 90,
-		/obj/item/storage/chewables/tobacco/bad = 60,
-		/obj/item/storage/chewables/tobacco = 90,
-		/obj/item/storage/chewables/tobacco/fine = 120,
-		/obj/item/storage/fancy/chewables/tobacco/nico = 150,
+		/obj/item/storage/fancy/cigarettes/rugged = 10,
+		/obj/item/storage/fancy/cigarettes = 15,
+		/obj/item/storage/fancy/cigarettes/dromedaryco = 25,
+		/obj/item/storage/fancy/cigarettes/nicotine = 30,
```
Comment (alsoandanswer) [inline code/game/machinery/vending_types.dm]: They should be _at least_ above 150-200 credits.

File: code/game/machinery/vending_types.dm [hunk 3]
```diff
@@ -345,23 +345,23 @@
...
+		/obj/item/storage/chewables/rollable/bad = 5,
+		/obj/item/storage/chewables/rollable = 10,
+		/obj/item/storage/chewables/rollable/fine = 15,
+		/obj/item/storage/chewables/rollable/nico = 20,
+		/obj/item/storage/chewables/tobacco/bad = 20,
+		/obj/item/storage/chewables/tobacco = 25,
+		/obj/item/storage/chewables/tobacco/fine = 40,
+		/obj/item/storage/fancy/chewables/tobacco/nico = 15,
```
Comment (alsoandanswer) [inline code/game/machinery/vending_types.dm]: The reason why it's so expensive is because it's literally the **pure** nicotine reagent inside.

Comment (alsoandanswer) [inline code/game/machinery/vending_types.dm]: Again, why? They're recent additions and aren't over 100 credits.

File: code/game/machinery/vending_types.dm (hunk 2 above)
Comment (ParadoxSpace) [reply code/game/machinery/vending_types.dm]: Absolutely not.

**Comment by alsoandanswer** [comment]
Can you **please** provide a more substantial reason other than "no" and "it must be lower because it's too expensive", because at that point it's just begging the question. Also, remember the fact that you can already spawn in *with* cigarettes on loadout, practically free.

Just because "my character needs cigarettes so cheaper is better IC'ly" isn't a valid argument either.

It's here for balance, so people won't clear out a cigarette machine, and makes sense IC'ly since tobacco is a commodity and luxury in a backwater station like Aurora.

The prices of cigarettes have **not** been a problem. At all. 
No ideas & suggestion thread in the forums, either.

**Comment by ParadoxSpace** [comment]
oh fuck. hang on.

**Pull request closed without merging**